src/import/ImportFFmpeg.cpp

```diff
@@ -255,24 +255,26 @@ class FFmpegImportFileHandle final : public ImportFileHandle
    }
 
 private:
-   std::shared_ptr<FFmpegFunctions> mFFmpeg;
+   // Construct this member first, so it is destroyed last, so the functions
```

Comment:
Well, it would be destroyed last anyway, as the member is first. But const - that is what I like a lot :)

Reply:
But note the important thing that the handle should have been constructed only after the possible reloading of libraries, right?